Src/AutoFixture/Kernel/FieldSpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool FieldIsCompatibleWithTargetType(object request)
 
         private bool FieldHasTargetName(object request)
         {
-            return ((FieldInfo)request).Name == this.targetName;
+            return ((FieldInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
[Clean Code](http://amzn.to/XCJi9X) explains why boolean flags are a bad idea.

Could you think of another way to vary comparison without introducing boolean flags?